js/private/js_binary.sh.tpl

```diff
@@ -138,13 +138,12 @@ function is_windows {
 # It helps to normalizes paths when running on Windows.
 #
 # Example:
-# C:/Users/XUser/_bazel_XUser/7q7kkv32/execroot/A/b/C -> /c/users/xuser/_bazel_xuser/7q7kkv32/execroot/a/b/c
+# C:/Users/XUser/_bazel_XUser/7q7kkv32/execroot/A/b/C -> /c/Users/XUser/_bazel_XUser/7q7kkv32/execroot/A/b/C
 function normalize_windows_path {
     # Apply the followings paths transformations to normalize paths on Windows
     # -process driver letter
     # -convert path separator
-    # -lowercase everything
-    sed -e 's#^\(.\):#/\L\1#' -e 's#\\#/#g' -e 's/[A-Z]/\L&/g' <<< "$1"
+    sed -e 's#^\(.\):#/\L\1#' -e 's#\\#/#g' <<< "$1"
```

Comment:
How did windows capitalization come into play with this issue?

Reply:
The replacement of `/MANIFEST` is simpler if we don't lowercase it. Not sure why it was there in the first place; that code came from rules_nodejs